### ISSUE TITLE
Невидимые AFK игроки

### ISSUE THREAD

**Issue by the-swank** [FWGS/xash3d#272]
Очень часто на базе можно заметить невидимых игроков, судя по всему это игроки в состоянии подключения к серверу

**Comment by mittorn** [comment]
Думаю, это должно исправляться изменениями в твоём PR, не так ли?

**Comment by the-swank** [comment]
Действительно, это из-за моего PR, мне почему-то показалось, что эта проблема есть на всех серверах

**Comment by the-swank** [comment]
Нет, мне таки не показалось, проблема проявляется в стоковой 0.19.x

**Comment by mittorn** [comment]
После отключения игрок остаётся па том же месте. В hl он отображается *

**Comment by a1batross** [comment]
Видать чем-то пофиксилось. Давно такого не видел.

**Comment by the-swank** [comment]
Поспрашивал у тех кто в cs16-client играет, пишут что есть на всех серверах
